piksi_multi_rtk_ros/src/piksi_multi_rtk_ros/piksi_multi.py

```diff
@@ -1565,7 +1565,7 @@ def cb_sbp_imu_aux(self, msg_raw, **metadata):
         self.gyro_scale = gyro_range * PiksiMulti.kGyroPrescale
 
         self.has_imu_scale = True
-        rospy.loginfo_once("Received IMU scale.")
+        rospy.loginfo("Received IMU scale.")
```

Comment:
Can you wrap the log info to make sure it is published only once?

```suggestion
        if not self.has_imu_scale:
            rospy.loginfo("Received IMU scale.")
        self.has_imu_scale = True  
```